src/store/widgets/reducers.ts

```diff
@@ -1,22 +1,28 @@
 import {
-  EDITOR_NEW_ANNOTATION,
+  EDITOR_ANNOTATION,
   EDITOR_VISIBLE_CHANGE,
-  MENU_WIDGET_CHANGE,
+  MENU_WIDGET_CHANGE, SET_EDITOR_SELECTION_RANGE,
   VIEWER_VISIBLE_CHANGE,
 } from './actions';
-import * as _ from "lodash";
+import * as _ from 'lodash';
```

Comment:
As a side note, if just a specific function is wanted (and we really care about final package size), it's better to write import as sth like `import _get from 'lodash/get'`, as then only the given fragment gets imported into our bundle.

If we just do a `import foo from 'lodash'`, then (usually) the whole package is injected into our final bundle, which often is not needed.

Reply:
Good advice :+1: